deprecation warnings all run together with no newlines

I also suggest using "Deprecation warning", or "Warning: ... is deprecated and will be removed in the next release; use ... instead".


```
google/go-cloud/samples/guestbook$ wire 
Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/blob/s3blob/s3blob.go:152:6: using struct literal to inject gocloud.dev/blob/s3blob.Options, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/blob/s3blob/s3blob.go:130:6: using struct literal to inject gocloud.dev/blob/s3blob.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/pubsub/awssnssqs/awssnssqs.go:421:6: using struct literal to inject gocloud.dev/pubsub/awssnssqs.SubscriptionOptions, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/pubsub/awssnssqs/awssnssqs.go:263:6: using struct literal to inject gocloud.dev/pubsub/awssnssqs.TopicOptions, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/pubsub/awssnssqs/awssnssqs.go:185:6: using struct literal to inject gocloud.dev/pubsub/awssnssqs.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/runtimevar/awsparamstore/awsparamstore.go:139:6: using struct literal to inject gocloud.dev/runtimevar/awsparamstore.Options, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/runtimevar/awsparamstore/awsparamstore.go:75:6: using struct literal to inject gocloud.dev/runtimevar/awsparamstore.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/secrets/awskms/kms.go:220:6: using struct literal to inject gocloud.dev/secrets/awskms.KeeperOptions, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/secrets/awskms/kms.go:110:6: using struct literal to inject gocloud.dev/secrets/awskms.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/aws/rds/rds.go:48:6: using struct literal to inject gocloud.dev/aws/rds.CertFetcher, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/server/server.go:56:6: using struct literal to inject gocloud.dev/server.Options, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/runtimevar/gcpruntimeconfig/gcpruntimeconfig.go:141:6: using struct literal to inject gocloud.dev/runtimevar/gcpruntimeconfig.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/secrets/gcpkms/kms.go:202:6: using struct literal to inject gocloud.dev/secrets/gcpkms.KeeperOptions, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/secrets/gcpkms/kms.go:113:6: using struct literal to inject gocloud.dev/secrets/gcpkms.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/pubsub/gcppubsub/gcppubsub.go:303:6: using struct literal to inject gocloud.dev/pubsub/gcppubsub.SubscriptionOptions, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/pubsub/gcppubsub/gcppubsub.go:227:6: using struct literal to inject gocloud.dev/pubsub/gcppubsub.TopicOptions, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/pubsub/gcppubsub/gcppubsub.go:158:6: using struct literal to inject gocloud.dev/pubsub/gcppubsub.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/blob/gcsblob/gcsblob.go:167:6: using struct literal to inject gocloud.dev/blob/gcsblob.Options, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.Deprecated: /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/blob/gcsblob/gcsblob.go:127:6: using struct literal to inject gocloud.dev/blob/gcsblob.URLOpener, use wire.Struct instead, see https://godoc.org/github.com/google/wire#Struct for more information.gocloud.dev/samples/guestbook: wrote /usr/local/google/home/rvangent/go/src/github.com/google/go-cloud/samples/guestbook/wire_gen.go

```

--- a/internal/wire/parse.go
+++ b/internal/wire/parse.go
@@ -753,9 +753,10 @@ func processStructLiteralProvider(fset *token.FileSet, typeName *types.TypeName)
 
 	pos := typeName.Pos()
 	fmt.Fprintf(os.Stderr,
-		"Deprecated: %v, see https://godoc.org/github.com/google/wire#Struct for more information.\n",
+		"Warning: %v, see https://godoc.org/github.com/google/wire#Struct for more information.\n",
 		notePosition(fset.Position(pos),
-			fmt.Errorf("using struct literal to inject %s, use wire.Struct instead", typeName.Type())))
+			fmt.Errorf("using struct literal to inject %s is deprecated and will be removed in the next release; use wire.Struct instead",
+				typeName.Type())))
 	provider := &Provider{
 		Pkg:      typeName.Pkg(),
 		Name:     typeName.Name(),

--- a/internal/wire/parse.go
+++ b/internal/wire/parse.go
@@ -753,7 +753,7 @@ func processStructLiteralProvider(fset *token.FileSet, typeName *types.TypeName)
 
 	pos := typeName.Pos()
 	fmt.Fprintf(os.Stderr,
-		"Deprecated: %v, see https://godoc.org/github.com/google/wire#Struct for more information.",
+		"Deprecated: %v, see https://godoc.org/github.com/google/wire#Struct for more information.\n",
 		notePosition(fset.Position(pos),
 			fmt.Errorf("using struct literal to inject %s, use wire.Struct instead", typeName.Type())))
 	provider := &Provider{

